CONTRIBUTING.md

```diff
@@ -52,6 +56,7 @@ ensure that your code complies with some standard style conventions.
   ```
   - Make sure that the tests pass
   ```
+  python setup.py install
   python setup.py test
```

Comment:
oh [there is](http://stackoverflow.com/questions/4734292/specifying-where-to-install-tests-require-dependencies-of-a-distribute-setupto#7747140)! :) just add `wget` to our `tests_require` :)